_templates/layout.html

```diff
@@ -0,0 +1,32 @@
+{% extends '!layout.html' %}
+
+{% block footer %}
+
+<div class="footer">
+    <p>© Copyright <a href="https://github.com/publiccodeyml/publiccode.yml/blob/main/AUTHORS.md">publiccode.yml authors</a> <span style="margin-left:8px; margin-right: 8px;">|</span> <a href="https://github.com/publiccodeyml/publiccode.yml">Github</a></p>
```

Comment:
minor fix to capitalisation

```suggestion
    <p>© Copyright <a href="https://github.com/publiccodeyml/publiccode.yml/blob/main/AUTHORS.md">publiccode.yml authors</a> <span style="margin-left:8px; margin-right: 8px;">|</span> <a href="https://github.com/publiccodeyml/publiccode.yml">GitHub</a></p>
```